modules/persistent-storage-csi-gcp-pd-encrypted-pv.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-gcp-pd.adoc
+
+[id="persistent-storage-csi-gcp-pd-encrypted-pv_{context}"]
+= Creating a custom-encrypted persistent volume
+
+When you create a `PersistentVolumeClaim` object, {product-title} provisions a new persistent volume (PV) and creates a `PersistentVolume` object. You can add a custom encryption key in Google Cloud Platform (GCP) to protect a PV in your cluster by encrypting the newly created PV.
+
+For encryption, the newly attached PV that you create uses customer-managed encryption keys (CMEK) on a cluster by using a new or existing Google Cloud Key Management Service (KMS) key.
+
+.Prerequisites
+* You are logged in to a running {product-title} cluster.
+* You have created a Cloud KMS key ring and key version.
+
+For more information about CMEK and Cloud KMS resources, see link:https://cloud.google.com/kubernetes-engine/docs/how-to/using-cmek[Using customer-managed encryption keys (CMEK)].
+
+.Procedure
+To create a custom-encrypted PV, complete the following steps:
+
+. Create a storage class with the Cloud KMS key. The following example enables dynamic provisioning of encrypted volumes:
++
+[source,yaml]
+--
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: csi-gce-pd-cmek
+provisioner: pd.csi.storage.gke.io
+volumeBindingMode: "WaitForFirstConsumer"
+allowVolumeExpansion: true
+parameters:
+  type: pd-standard
+  disk-encryption-kms-key: projects/<key-project-id>/locations/<location>/keyRings/<key-ring>/cryptoKeys/<key> <1>
```

Comment:
The annotation for this is way over to the right, hidden by a scroll bar. I don't know if there is a better solution but figured I'd mention. In theory, the person reading this should look all the way into the scrolled area anyway, right? :slightly_smiling_face:

Reply:
Yeah, I didn't love that either and not sure of a better suggestion. I agree they should be scrolling if reading.